.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "astropy_helpers"]
 	path = astropy_helpers
-	url = https://github.com/Cadair/astropy-helpers.git
+	url = https://github.com/astropy/astropy-helpers.git
```

Comment:
THIS HAS BEEN SO MUCH PAIN FOR ME.

Reply:
oh man didn't we back this out already :scream_cat:

Reply:
Yeah, just on this branch.